### DISCUSSION TITLE
Is there an event listsner for recent activities?

### DISCUSSION THREAD

**Discussion by nathanlochunho** [cwendt94/espn-api#385] [Q&A]
Am building a program that sends email when there is a new activity in my league. Wondering if there is an event listsner that can handle this problem or whats the best way of doing it?
Thanks alot!

**Comment by nathanlochunho** [comment]
I have done it, if anyone interested, I can share the code.

**Reply by cwendt94** [reply]
You can get all recent activity from `recent_activity` function in the league but you would have to add your own logic to know if its new or not/poll the data.

I would love to see what you did!
